src/archiefvernietigingscomponent/destruction/tasks.py

```diff
@@ -99,13 +99,21 @@ def process_list_item(list_item_id):
             template="destruction/logs/item_destruction_succeeded.txt",
             extra_data={"zaak": zaak["identificatie"]},
         )
+        try:
+            resultaat = fetch_resultaat(zaak["resultaat"])
```

Comment:
I think `resultaat` should be already destroyed when you fetch it

Reply:
Good catch!